Updated the project

nisrulz opened the pull request:
Updated
- gitignore rules
- android gradle tools plugin to v2.2.2
- compile sdk version to v25
- build tools version to v25.0.1
- target sdk version to 25
- support lib version to v25.0.1
- butterknife to v8.4.0
- timber to v4.3.1
- retrofit to v2.1.0
- gradle wrapper to v2.14.1

nisrulz committed:
[*] updated
- gitignore rules
- android gradle tools plugin to v2.2.2
- compile sdk version to v25
- build tools version to v25.0.1
- target sdk version to 25
- support lib version to v25.0.1
- butterknife to v8.4.0
- timber to v4.3.1
- retrofit to v2.1.0
- gradle wrapper to v2.14.1

carlosjs23 reviewed (commented):
thanks you for keep update the boilerplate.

nisrulz committed:
[*] updated
- android gradle plugin to v2.2.3
- support lib version to v25.1.0
-  timber lib to v4.4.0

nisrulz commented:
Further updated
- android gradle plugin to v2.2.3
- support lib version to v25.1.0
- timber lib to v4.4.0

nisrulz commented:
Any updates on this PR regarding merging ?
